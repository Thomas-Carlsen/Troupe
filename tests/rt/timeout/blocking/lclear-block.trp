(* this program blocks in the receive, because receive without explicit bounds 
   uses [pc,pc] as its low/high bounds *)

let val main = self ()
    val _ = spawn (fn () => 
                     if (1 raisedTo {alice}) then send (main, 42) 
                     else ())    
    val _ = receive [ hn x => print x ] 
in ()
end
 