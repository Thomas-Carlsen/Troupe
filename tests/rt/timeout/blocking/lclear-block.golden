2020-02-09T01:35:02.510Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
